libbulletc/src/src/LinearMath/btVector3.cpp

```diff
@@ -827,7 +827,7 @@ long _mindot_large( const float *vv, const float *vec, unsigned long count, floa
 #include <sys/types.h>
 
 #ifdef ANDROID
-  #include "NDK/sources/android/cpufeatures/cpu-features.h"
+  #include "cpu-features.h"
```

Comment:
Fixed compilation error for Android: `../../src/src/LinearMath/btCpuFeatureUtility.h:19:12: fatal error: 'NDK/sources/android/cpufeatures/cpu-features.h' file not found`

Notes:
Explicit defining `NDK` through command line args like this:
```
LOCAL_CFLAGS := -DNDK="/android/ndk/23.0.7599858"
```

not solved an issue.

Useful macros for checking NDK value:
```
#define XSTR(x) STR(x)
#define STR(x) #x
 
#pragma message "The value of NDK: " XSTR(NDK)
```